README.md

```diff
@@ -0,0 +1,15 @@
+# Phonebook
+Mozilla's internal phonebook app.
+
+## Installation
+* Clone, host on a local, PHP-enabled server, run.
+* Optional: copy ``config-local.php-dist`` to ``config-local.php`` and tweak as needed
+
+*Note:* The app requires LDAP server access. You probably need a VPN connection up and running.
+
+## Contributing
+
+If you feel so inclined, feel free to contribute:
```

Comment:
"Feel" twice in a sentence, also not too straightforward for ESL folks :wink: 

`To contribute, file bugs in Bugzilla and use GitHub for code review:`